src/seedu/addressbook/data/AddressBook.java

```diff
@@ -25,12 +27,15 @@
     private final UniquePersonList allPersons;
     private final UniqueTagList allTags; // can contain tags not attached to any person
 
+    public ArrayList<Tagging> listOfTaggingRecords;
```

Comment:
Good, this is a correct place to put the list based on the UML.. Any reason for the `public`?